recipes/cdecimal/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cdecimal" %}
+{% set version = "2.3" %}
+{% set sha256 = "d737cbe43ed1f6ad9874fb86c3db1e9bbe20c0c750868fde5be3f379ade83d8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.bytereef.org/software/mpdecimal/releases/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not py2k]
```

Comment:
I would prefer to not use the negative :smile: 

It is easier to read `skip: True  # [py3k]`